dev/tests/fixtures/docfx/NewClient/V1.Client.SecretManagerServiceClient.yml

```diff
@@ -10,6 +10,7 @@ items:
 
       This class is currently experimental and may be subject to changes.
     type: class
+    namespace: 'Google \ Cloud \ SecretManager \ V1 \ Client'
```

Comment:
Based on the example from https://github.com/googleapis/doc-pipeline/issues/478, I think the namespace should have the `SecretManagerServiceClient` (maybe in href) added at the end - do you want to exclude the current class?

Reply:
We also found that we can’t add HTML here so we are going to skip this for now (and hopefully add it later)